### PR TITLE
RWKV: fix mask warning typo

### PR DESCRIPTION
**Pull request by RobinKa**
`RwKvModel.forward` is supposed to warn if `attention_mask` is passed. Right now the check is `attention_mask is None` which I assume was supposed to be `attention_mask is not None`, which this PR fixes. @gante

### CODE REVIEW

**Review by ArthurZucker — APPROVED** [review]
Thanks!
